src/bosh-google-cpi/action/concrete_factory.go

```diff
@@ -127,22 +127,12 @@ func (f ConcreteFactory) Create(method string, ctx map[string]interface{}) (Acti
 		f.logger,
 	)
 
-	// Choose the correct registry.Client based on the
-	// value of ClientOptions.UseGCEMetadata
-	var registryClient registry.Client
-	switch f.cfg.Cloud.Properties.Registry.UseGCEMetadata {
-	case true:
-		registryClient = registry.NewMetadataClient(
-			googleClient,
-			f.cfg.Cloud.Properties.Registry,
-			f.logger,
-		)
-	default:
```

Comment:
Is there a reason to remove support for the HTTP client?

Reply:
Although this isn't "deadcode" per se, but the release itself offers no way for an operator to use it.

The configurable option `registry.use_gce_metadata` in `jobs/google_cpi/spec` doesn't get used in the `cpi.json.erb`; it gets defaulted to `true`.

I don't recall tests using this either, please correct me if I'm mistaken.